conf/script/prov-m0-reset

```diff
@@ -130,10 +152,32 @@ fi
 [[ -b $lvolume ]] || die "meta-data volume $lvolume is not available"
 [[ -b $rvolume ]] || die "meta-data volume $rvolume is not available"
 
+export PATH=$PATH:/opt/seagate/cortx/hare/bin
+
+# This function is duplicated from build-ha-io script.
+# TODO: Remove the function definition once the common functions are available.
+prepare_var_motr_dirs() {
+    log "${FUNCNAME[0]}: make sure there are no stale mounts hanging..."
+    mountpoint -q /var/motr1 && sudo umount /var/motr1 || true
```

Comment:
What if this script is run from right node ?

Reply:
Yes, for reset it was assumed that is the node is not working then that is the first problem that we need to work on. But its still a valid case but again I will make this change for reset separately as there is other related code for reset that needs to be updated for multinode case.
I have updated mkfs for now which is being used. Do you want to fix reset in this patch too?

Reply:
Also another reason for reset to be executed from primary node as it runs hctl bootstrap which requires some files which are presently only created on primary node.